utils.js

```diff
@@ -114,6 +114,46 @@ const getPathId = async ({ apig, apiId, endpoint }) => {
   return endpointFound ? endpointFound.id : null
 }
 
+const enableCORS = ({ endpoints }) => {
+  const defaultResponse = [{
+    code: 200,
+    headers: {
```

Comment:
Should these be made overwrite-able?

Reply:
they are, those are just defaults, you can include them in the yml file to overwrite